hack/00-osd-managed-cluster-config-integration.yaml.tmpl

```
@@ -3559,11 +3561,63 @@ objects:
       managed.openshift.io/gitHash: ${IMAGE_TAG}
       managed.openshift.io/gitRepoName: ${REPO_NAME}
       managed.openshift.io/osd: 'true'
-    name: kubelet-config
+    name: kubelet-config-version-major-minor
+  spec:
+    clusterDeploymentSelector:
+      matchLabels:
+        api.openshift.com/managed: 'true'
+      matchExpressions:
+      - key: hive.openshift.io/version-major-minor
+        operator: NotIn
+        values:
+        - '4.8'
+        - '4.9'
+    resourceApplyMode: Upsert
+    patches:
+    - kind: MachineConfigPool
+      apiVersion: machineconfiguration.openshift.io/v1
+      name: worker
+      applyMode: AlwaysApply
+      patch: '{ "metadata": {"labels": {"managed.openshift.io/custom-kubelet":""}
```

```suggestion
      patch: '{ "metadata": {"labels": {"managed.openshift.io/system-reserved-worker":""}
```

This needs to change no?

---

Nope. This is the existing configuration, but it's being renamed/duplicated because of the `OR` condition.